layouts/error.vue

```diff
@@ -1,20 +1,21 @@
 <template>
   <v-app dark>
     <h1 v-if="error.statusCode === 404">
-      {{ pageNotFound }}
+      <Error404 />
     </h1>
     <h1 v-else>
       {{ otherError }}
     </h1>
-    <NuxtLink to="/">
-      Home page
-    </NuxtLink>
   </v-app>
 </template>
 
 <script>
+import Error404 from './errors/404.vue'
```

Comment:
Importing components is not needed. Just use them in template

Reply:
Fixed on ad45b59e259609c193f3ed27044db1eb94e93552